framework/src/bundle/BundleResourceContainer.cpp

```diff
@@ -265,11 +260,17 @@ bool BundleResourceContainer::Matches(const std::string& name,
   return true;
 }
 
-void BundleResourceContainer::OpenContainer()
+void BundleResourceContainer::OpenContainer() const
```

Comment:
this no longer only opens the container. I suggest a method name change to describe the intent. suggestion: `OpenAndInitializeContainer`